fetcher/fetcher.go

```diff
@@ -0,0 +1,102 @@
+package fetcher
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+var DefaultRepos = []string{
+	"https://Repo.maven.apache.org/maven2",
+	"https://dl.google.com/dl/android/maven2",
+	"https://repository.sonatype.org/content/groups/sonatype-public-grid",
+}
```

Comment:
This is a bit of personal taste, but really you should have lists that your application depends on in a variable, it makes it easier for code inspection.

Reply:
Yeah, that was just a leftover form changes.